src/components/YoutubeVideoPlayer/YoutubeVideoPlayer.component.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  StyledContainer,
+  StyledVideoPlayer,
+  StyledVideoDetails,
+  StyledVideoTitle,
+} from './YoutubeVideoPlayer.styles';
+
+function VideoPlayer(props) {
+  const { id, snippet } = props.videoInfo;
+  const { title, description } = snippet;
```

Comment:
Same here, no need to pass the whole videoInfo object
```suggestion
  const { title, description, id } = props;
```